AnnoyancesFilter/sections/widgets.txt

```diff
@@ -552,6 +552,9 @@ subbota.ua##aside.sidebar > div[id^="custom_html-"]
 !
 ! Donation widgets (which are really annoying only)
 !
+hani.co.kr##.bn-promotion-support
+m.hani.co.kr##.support-content
+m.hani.co.kr##.bn-promotion-support
```

Comment:
Remove this rule - it's a duplicate.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/87557/commits/cf99e173fdf278fa0d624bb52c14f2e1c4a056b3